core-ui/src/main/java/com/anytypeio/anytype/core_ui/views/DesignSystemButtons.kt

```diff
@@ -563,4 +569,30 @@ fun MyWarningButton() {
             .fillMaxWidth()
             .padding(start = 16.dp, end = 16.dp)
     )
+}
+
+class ButtonPrimarySmallIcon @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+) : LinearLayout(context, attrs) {
+
+    private lateinit var button: TextView
+    private lateinit var icon: ImageView
+
+    init {
+        init(context)
```

Comment:
init inside init looks a little bit weird, maybe let's name it`setup`?

Reply:
renamed to setup